### DISCUSSION TITLE
Fine tuning volume

### DISCUSSION THREAD

**Discussion by Findus89** [advplyr/audiobookshelf#1685] [Ideas]
I am a long time user of Smart audiobook player but we're now trying to swap over. The only function i miss in Abs is the ability to fine-tune the volume, like lower it and increase it. I usually have an earbud and listen when in bed. It's really nice to be able to lower the overall volume and then adjust it with the regular volume control for the perfect level.

**Comment by WarHawk8080** [comment]
I put an app that does it for me..."super volume booster" but I bet there are hundreds of em like this out there

I wonder if the entire library could be "equalized" so some books that are too low volume can be brought up to a specific db level, and loud ones brought down

I did find an online one https://products.aspose.app/audio/change-volume/m4b
But having the option IN the program to set all the books in the entire library to a specific db level would be SWEET!
